src/components/ReleaseService/ReleasePlan/TriggerRelease/TriggerReleaseForm.tsx

```diff
@@ -124,6 +124,7 @@ export const TriggerReleaseForm: React.FC<Props> = ({
           <TextColumnField
             name="references"
             label="References"
+            helpText="Any URLs entered must be separated by commas."
```

Comment:
hmm, that's not the case anymore. Not long ago we updated the behavior here, so the user will add one reference per input.

see example below:

https://github.com/user-attachments/assets/f76c17b6-6eec-41f5-aec5-258a4a91b577

I'd suggest to reach out to Pankaj and show him how it looks like, so we can have a different `helpText` :)

something like `"Add any references related to the release"`

Reply:
does comma separation still work? 

If I do ```ref1, ref2``` and do + does it add 2 of them? If not I would change text completly